### PR TITLE
added vantage.js

### PR DESCRIPTION
**Pull request by dthree**

### CODE REVIEW

**Comment by sindresorhus** [comment]
New entries should be added to the bottom of the section.

File: readme.md [hunk 1]
```diff
@@ -76,6 +76,7 @@ Inspired by the [awesome](https://github.com/sindresorhus/awesome) list thing.
 
 - [pageres](https://github.com/sindresorhus/pageres) - Capture website screenshots.
 - [trash](https://github.com/sindresorhus/trash) - A safer alternative to `rm`.
+- [vantage](https://github.com/dthree/vantage) - Distributed, realtime CLI for live Node apps.
```
Comment (sindresorhus) [inline readme.md]: Node => Node.js

**Comment by dthree** [comment]
Thanks and tracking. Wasn't sure where to put it. All fixed now.

**Comment by sindresorhus** [comment]
@sboudrias Fyi, this uses Inquirer ;)

**Comment by dthree** [comment]
Inquirer and Commander are the only reason this app qualifies as awesome :)

**Comment by sindresorhus** [comment]
You might want to reconsider your use of `colors` though. See: https://github.com/chalk/chalk

**Comment by dthree** [comment]
Thanks for the advice. Was looking at that a few weeks ago - I wasn't sure if native string extensions was a big argument in the real world, or just theoretically. If it actually turns off people, I can totally change it.

**Comment by sindresorhus** [comment]
It is. Especially in Node, since eg. a sub-sub-sub dependency extending a native prototype will affect all running dependencies.

**Comment by dthree** [comment]
Good point. I'll definitely be doing that then.

Anyway, thanks for accepting this PR! Hope you like it otherwise. 

David

**Comment by sindresorhus** [comment]
Yup, super cool idea :)

**Comment by dthree** [comment]
Thanks :)

**Comment by dthree** [comment]
@sindresorhus thought I'd let you know that I finished ditching `colors` and implemented `chalk` into Vantage.js. I also noticed I happened to use your `strip-ansi` module as well lol. With that and a bunch of other fixes, Vantage hit 1.0 today :)

**Comment by sindresorhus** [comment]
Awesome! :)

**Pull request closed without merging**